set.mm

```diff
@@ -99138,6 +99172,25 @@ is that it denies the existence of a set containing itself ( ~ elirrv ).
     ( cvv wcel wa wn cep wfr zfregfr efrn2lp mpan elex anim12i con3i pm2.61i )
     ACDZBCDZEZABDZBADZEZFZCGHRUBCICABJKUARSPTQABLBALMNO $.
 
+  ${
+    $d x y $.
+    $( The membership (epsilon) relation and its converse are disjoint.
+       Variable-free version of ~ en2lp .  (Proposed by BJ, 18-Jun-2022.)
+       (Contributed by AV, 19-Jun-2022.) $)
+    cnvepnep $p |- ( `' _E i^i _E ) = (/) $=
+      ( vy vx cep ccnv cin wel wa copab c0 df-eprel cnveqi cnvopab eqtri inopab
+      ineq12i wceq wn wal cv en2lp gen2 opab0 mpbir ) CDZCEZABFZBAFZGZBAHZIUEUF
+      BAHZUGBAHZEUIUDUJCUKUDUFABHZDUJCULABJKUFABLMBAJOUFUGBANMUIIPUHQZARBRUMBAA
+      SBSTUAUHBAUBUCM $.
+
+    $( The membership (epsilon) relation is asymmetric.  (Contributed by AV,
+       18-Jun-2022.) $)
+    epasym $p |- `' _E =/= _E $=
+      ( cep ccnv cin c0 wceq wa wb wne cnvepnep disjeq0 wi epn0 eqneqall adantl
```

Comment:
"asymmetric" is the property proved in ~cnvepnep.

~epasym simply is "not symmetric" (which is not a very important property)

Reply:
Comments and labels adjusted.